cranelift/codegen/src/isa/x64/inst/external.rs

```diff
@@ -98,27 +91,6 @@ impl From<Writable<Reg>> for asm::Gpr<WritableGpr> {
     }
 }
 
-// For Winch ergonomics.
-impl From<PairedGpr> for asm::Gpr<PairedGpr> {
-    fn from(pair: PairedGpr) -> Self {
-        Self::new(pair)
-    }
-}
-
-// For Winch ergonomics.
-impl From<PairedGpr> for asm::GprMem<PairedGpr, Gpr> {
-    fn from(pair: PairedGpr) -> Self {
-        Self::Gpr(pair)
-    }
-}
-
-// For Winch ergonomics.
-impl From<WritableGpr> for asm::Gpr<WritableGpr> {
-    fn from(gpr: WritableGpr) -> Self {
-        Self::new(gpr)
-    }
-}
```

Comment:
Thanks for this. Putting the `From` over on the assembler side should be clearer!